task/sql-tasks.js

```diff
@@ -280,7 +463,27 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    select
+	distinct
+            Customers.CompanyName,
+            Products.ProductName,
+            OrderDetails.UnitPrice as "PricePerItem"
+        from Customers 
+        inner join Orders ON Customers.CustomerID=Orders.CustomerID
+        inner join OrderDetails ON Orders.OrderID=OrderDetails.OrderID
+        inner join Products ON OrderDetails.ProductID=Products.ProductID
+        WHERE OrderDetails.UnitPrice = (select

```

Comment:
company name doesn't uniquely identify customer. also this column doesn't have an index so it's also slow.
also try to use group by customer id and join subquery on unit price and customer id to speed up query